companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -2221,13 +2257,19 @@ class ModuleField: public TransformedField {
       else if (module.protocol == PULSES_MULTIMODULE) {
         module.rfProtocol = module.multi.rfProtocol & 0x0F;
       }
+      else if (module.protocol == PULSES_AFHDS3) {
+        module.rfProtocol = module.subType;
```

Comment:
We will merge rfProtocol and subType in 2.4
I suggest you use subType for this module

Reply:
Done